src/main/java/io/github/bookcrawler/entities/BookInfo.java

```diff
@@ -37,16 +57,20 @@ public String getDescription() {
         return description;
     }
 
+
     public String getPrice() {
         return price;
     }
 
+    public String getUrl() {
+        return url;
+    }
+
     public String getLibrary() {
         return library;
     }
 
-    public String getUrl() {
-        return url;
+    public Set<Tag> getTags() {
+        return tags;
     }
-
 }
```

Comment:
You can use lombok annotation to avoid creating getters and setters and constructore :facepunch: